src/App.css

```diff
@@ -1,9 +1,32 @@
 .App {
-  display: block;
+  display: flex;
+  flex-direction: column;
   text-align: center;
   background-color: #222;
   color: #fff;
   height: 100vh;
 
   font-family: Arial, Helvetica, sans-serif;
 }
+
+.App__Content {
+  flex-grow: 1;
+}
+
+.ExperimentLinks {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: center;
+}
+
+.ExperimentLinks__Link {
+  margin: 1rem 0.5rem;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #888;
+}
+
+.ExperimentLinks__Link.active {
```

Comment:
Alas, couldn't go full BEM here since react router flags the active element by adding an `active` class.

Reply:
No worries. My CSS layer is utter crap right now. Just the bare necessities. The simple bare necessities...